sys-auth/sssd/sssd-1.13.1-r1.ebuild

```diff
@@ -14,7 +14,7 @@ SRC_URI="http://fedorahosted.org/released/${PN}/${P}.tar.gz"
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86"
```

Comment:
how did this work before if it was ~amd64?

also, this should be 'arm64', arm is 32 bit.